ext/spl/spl_array.c

```diff
@@ -311,7 +311,7 @@ static zval *spl_array_get_dimension_ptr(int check_inherited, spl_array_object *
 	case IS_STRING:
 	   offset_key = Z_STR_P(offset);
 fetch_dim_string:
-		retval = zend_symtable_find(ht, offset_key);
+		retval = zend_hash_find(ht, offset_key);
```

Comment:
Doesn't think break access for normal arrays?

Reply:
Oh, indeed! Thanks.